lib/psulib_blacklight/cleaner.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module PsulibBlacklight
+  class Cleaner
```

Comment:
@awead Would it be more clear if we rename this class now that it also have the loading fixtures method? I do not have suggestion though, you are way more creative for coming up with nicer names :)

Reply:
@banukutlu DataManager? We have a SolrManager....

Reply:
yeas 👍🏻

Reply:
All fixed up!